packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -1032,7 +1040,7 @@ https://developer.litprotocol.com/v3/sdk/wallets/auth-methods/#auth-method-scope
       return scope;
     });
 
-    const authId = await getAuthIdByAuthMethod(authMethod);
+    const _authId = authId ?? (await getAuthIdByAuthMethod(authMethod));
```

Comment:
What happens with an empty authMethod/authId?

Reply:
You can't have empty `authMethod` for this function, only `authId` is optional